admiral/envs/components/examples/fighting_teams.py

```diff
@@ -2,60 +2,127 @@
 from matplotlib import pyplot as plt
 import numpy as np
 
+# Import all the features that we need from the simulation components
```

Comment:
Remove this example since we don't use it.

Reply:
We do not train with it, but it's nice to have hear as part of the components examples suite